src/x11_init.c

```diff
@@ -492,8 +502,7 @@ static GLFWbool initExtensions(void)
 
     if (_glfw.x11.randr.available)
     {
-        XRRScreenResources* sr = XRRGetScreenResources(_glfw.x11.display,
-                                                       _glfw.x11.root);
+        XRRScreenResources* sr = _glfwXRRGetScreenResources();
```

Comment:
This invalid call emits no warnings because of the the bad declaration below. It will pull its arguments from random garbage on the stack and likely segfault as the first parameter is a pointer.